projects/packages/videopress/src/client/state/resolvers.js

```diff
@@ -18,8 +18,6 @@ import {
 	SET_LOCAL_VIDEOS_QUERY,
 	WP_REST_API_USERS_ENDPOINT,
 	WP_REST_API_VIDEOPRESS_PLAYBACK_TOKEN_ENDPOINT,
-	VIDEO_PRIVACY_LEVELS,
-	VIDEO_PRIVACY_LEVEL_PRIVATE,
```

Comment:
should we remove these constants, too?

Reply:
Remove the import? Yes, because they are not being used in any other place of the file.